doc/user/polyglot.md

```diff
@@ -0,0 +1,92 @@
+# Polyglot Programming
+
+## Evaluating code in foreign languages
+
+`Polyglot.eval(id, string)` executes code in a foreign language identified by
+its ID.
+
+`Polyglot.eval_file(id, path)` executes code in a foreign language from a file,
+identified by its language ID.
+
+`Polyglot.eval_file(path)` executes code in a foreign language from a file,
+automatically determining the language.
+
+## Exporting Ruby objects to foreign languages
+
+`Polyglot.export(name, value)` exports a value with a given name.
+
+`Polyglot.export_method(name)` exports a method, defined in the top-level
+object.
+
+## Importing foreign objects to Ruby
+
+`Polyglot.import(name)` imports and returns a value with a given name.
+
+`Polyglot.import_method(name)` imports a method with a given name, and defines
+it in the top-level object.
```

Comment:
To be more precise, but maybe too much details:
`imports an IS_EXECUTABLE value, and defines it as a top-level method`.

Reply:
Adding 'which should be `IS_EXECUTABLE`'